app/addons/documents/mango/mango.components.react.jsx

```diff
@@ -129,7 +129,7 @@ function (app, FauxtonAPI, React, Stores, Actions,
       var url = '#/' + FauxtonAPI.urls('allDocs', 'app', this.props.dbName, '');
 
       return (
-        <div className="editor-wrapper span5 scrollable">
+        <div className="editor-wrapper">
```

Comment:
Actually not yet. The index-editor doesn't use two_pane yet, so the flexbox refactor will change that in a later commit.